sync2jira/upstream_issue.py

```diff
@@ -188,6 +188,13 @@ def github_issues(upstream, config):
         .get('github', {})\
         .get(upstream, {})
 
+    if 'labels' in _filter:
+        # We have to flatten the labels list to a comma-separated string
+        expected = _filter['labels']
+        if not isinstance(expected,str):
+            assert isinstance(expected,list)
```

Comment:
I think we are trying to work toward a more robust mode of operation for `sync2jira`, so having it die suddenly from an assertion failure would be undesirable.  I don't know enough to suggest what _should_ happen here (@ralphbean? @Zyzyx?), but, if the value is neither a `str` nor a list of `str`'s, perhaps it should be replaced with an empty string.

Also, I have to ask, why is this level of paranoia warranted for issues and not for PRs?  (The code you added  `sync2jira/upstream_pr.py` line 147 doesn't do the type checks.)

Reply:
Ah sorry, I actually didn't mean to push this - I was just testing this locally :sweat_smile:

Reply:
Removed this, now I always assume we have a list if the `labels` key exists.